src/components/Workspace/index.js

```diff
@@ -126,6 +158,7 @@ Workspace.propTypes = {
     xxs: PropTypes.number,
   }),
   rowHeight: PropTypes.number,
+  rows: PropTypes.number,
```

Comment:
Suggest adding comments on these new properties - such as what they do for the layout.

I wish the original author had commented the original properties, but going forward we should add comments as new properties are added.